File: 1791-find-center-of-star-graph/1791-find-center-of-star-graph.cpp
class Solution {
public:
    int findCenter(vector<vector<int>>& edges) {
        map<unsigned int, unsigned int> degree;
        for(unsigned int i=0;i<2;i++){
            degree[edges[i][0]]++;
            degree[edges[i][1]]++;
        }
        
        for(auto i:degree){
            if(i.second!=1){
                return i.first;
            }
        }
        
        return 0;
    }
};

File: 1791-find-center-of-star-graph/1791-find-center-of-star-graph.py
class Solution:
    def findCenter(self, edges: List[List[int]]) -> int:
        adj = defaultdict(list)
        
        for u,v in edges:
            adj[u].append(v)
            adj[v].append(u)
        for i in adj:
            if len(adj[i])>1:
                return i
            